doc/contributing/releases.md

```diff
@@ -1217,8 +1217,13 @@ the releaser, these must be kept in sync with `main`.
 The `vN.x` and `vN.x-staging` branches must be kept in sync with one another
 up until the date of the release.
 
-The TSC should be informed of any `SEMVER-MAJOR` commits that land within one
-month of the release.
+If a `SEMVER-MAJOR` pull requests lands on the default branch within one month prior to
+the major release date, it must not be included on the new major staging branch, unless there is consensus from the Node.js releasers
```

Comment:
Maybe adding `baking-for-lts` label helps releasing it by mistake?

Reply:
I think that will cause confusion -- that label is for LTS releases and not all semver majors become LTS.

Reply:
We don't need to remember if we sync a month earlier as it will be equal to `main`.